dracut/30ignition/ignition-diskful.target

```diff
@@ -0,0 +1,16 @@
+# This target contains Ignition units that should only run when we have
```

Comment:
others possibles: `ignition-withharddisk.target` `ignition-bootedfromdisk.target` blah blah blah :) 

> `diskful` being the opposite of `diskless` slightly_smiling_face

You know this might sound dumb, but putting that exact word in the description of this target unit file might actually help it click for people. 

*i.e. when we're not running diskless from a live image out of RAM*

Reply:
> You know this might sound dumb, but putting that exact word in the description of this target unit file might actually help it click for people.

Good point.  Updated.